packages/babel-plugin-bugfix-v8-spread-parameters-in-optional-chaining/src/index.ts

```diff
@@ -7,8 +7,8 @@ import type * as t from "@babel/types";
 export default declare(api => {
   api.assertVersion(7);
 
-  const noDocumentAll = api.assumption("noDocumentAll");
-  const pureGetters = api.assumption("pureGetters");
+  const noDocumentAll = (api.assumption("noDocumentAll") ?? false) as boolean;
```

Comment:
This is https://github.com/microsoft/TypeScript/issues/40359. TS currently can not narrow down `(boolean | void) ?? boolean` to `boolean`